Description: Stop forcing SSE and x86 compiler options in the generic target.
Author: Adam Conrad <adconrad@ubuntu.com>
Forwarded: no
---
 src/CMakeLists.txt |    5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

--- zynaddsubfx.orig/src/CMakeLists.txt
+++ zynaddsubfx/src/CMakeLists.txt
@@ -116,8 +116,8 @@ set (BuildOptions_X86_64Core2
 )
 
 set (BuildOptionsBasic
-    "-O3 -msse -msse2 -mfpmath=sse -ffast-math -fomit-frame-pointer"
-    CACHE STRING "basic X86 complier options"
+    "-O3 -ffast-math -fomit-frame-pointer"
+    CACHE STRING "basic complier options"
 )
 
 set (BuildOptionsDebug
@@ -173,7 +173,6 @@ endif()
 include_directories(${ZLIB_INCLUDE_DIRS} ${MXML_INCLUDE_DIRS})
 
 add_definitions(
-	 -DASM_F2I_YES
 	 -g #TODO #todo put in a better location
 	 -Wall
 	 -Wextra
